Extensions/dnSpy.Analyzer/TreeNodes/ClassInterfaceSubtypedByNode.cs

```diff
@@ -0,0 +1,67 @@
+/*
+    Copyright (C) 2014-2024 mitch.capper+dns@gmail.com
+
+    This file is part of dnSpy
+
+    dnSpy is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    dnSpy is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with dnSpy.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using dnlib.DotNet;
+using dnSpy.Analyzer.Properties;
+using dnSpy.Contracts.Decompiler;
+using dnSpy.Contracts.Text;
+
+namespace dnSpy.Analyzer.TreeNodes {
+	sealed class ClassInterfaceSubtypedByNode : SearchNode {
+		readonly TypeDef analyzedType;
+
+		public ClassInterfaceSubtypedByNode(TypeDef analyzedType) =>
+			this.analyzedType = analyzedType ?? throw new ArgumentNullException(nameof(analyzedType));
+
+		protected override void Write(ITextColorWriter output, IDecompiler decompiler) =>
+			output.Write(BoxedTextColor.Text, dnSpy_Analyzer_Resources.SubtypedByTreeNode);
+
+		protected override IEnumerable<AnalyzerTreeNodeData> FetchChildren(CancellationToken ct) {
+			var analyzer = new ScopedWhereUsedAnalyzer<AnalyzerTreeNodeData>(Context.DocumentService, analyzedType, FindReferencesInType);
+			return analyzer.PerformAnalysis(ct);
+		}
+
+		IEnumerable<AnalyzerTreeNodeData> FindReferencesInType(TypeDef type) {
+			if (analyzedType.IsInterface && type.HasInterfaces) {
+				foreach (var iface in type.Interfaces) {
+					if (IsEqual(iface.Interface)) {
+						yield return new TypeNode(type) { Context = Context };
+						break;
+					}
+				}
+
+				yield break;
+			}
+
+			if (type.IsEnum || !type.IsClass || type.IsSealed)

```

Comment:
Sealed types can still have base types. My original comment was to change the `CanShow` method since `sealed` types cannot be inherited from and will never have subtypes. The check of `|| type.IsSealed` should be removed here and the CanShow method should be modified to return false when the type is sealed.